frontend/views/containers/contributions/IncomeDetails.vue

```diff
@@ -1,79 +1,76 @@
 <template lang='pug'>
-modal-base-template(ref='modal')
-  .wrapper-container
-    .c-content
-      i18n.is-title-2.c-title(tag='h2') Income Details
+modal-base-template(ref='modal' :fullscreen='true')
+  .c-content
+    i18n.is-title-2.c-title(tag='h2') Income Details
```

Comment:
NOTE: I replaced this extra wrapper with the new modal prop fullscreen. The indentation changed! That's why this huge diff..

Just review the `group-pledges-graph` component, that is the real change.

Reply:
There is already 2 kind of modals, one full screen and one not. Is this a 3rd kind?

Reply:
As far as I understand, we don't have a "real" fullscreen modal. To implement a fullscreen modal we have been following different approaches:
1- create a wrapper child of `modal-base-template` and style it, adding the dimensions and the missing `background-color: $general_2`. Examples: this modal, PayGroupHistory, etc...
2 - Style `.modal` "indirectly". I say indirectly because it's not explicit that it's a class inside `modal-base-template`, for example `CreateGroup.vue` or `Join.vue`. There we also manually add the `background-color`. 
3 - Add that same bg to some elements inside the modal, for example, `SettingsWrapper` (the user settings modal), where we add the bg to its sidebar and its main.

And, no matter the approach, all of these modals are incorrectly styled because the paddings and the button style (having ESC) are wrong. 
For those reasons, I decided to explicit add a `fullscreen` prop to `modal-base-template` that deal with that for us. No need to add the bg color. Paddings are fixed and close button is styled correctly.

Does this make sense?